templates/quickstart-vpc-for-atlassian-services.yaml

```diff
@@ -160,12 +172,30 @@ Resources:
   VPCStack:
     Type: AWS::CloudFormation::Stack
     Properties:
-      TemplateURL: !Sub
-        - https://${QSS3BucketName}.${QSS3Region}.amazonaws.com/${QSS3KeyPrefix}submodules/quickstart-aws-vpc/templates/aws-vpc.template
-        - QSS3Region: !If
-            - GovCloudCondition
-            - s3-us-gov-west-1
-            - s3
+      # Historically this template has never been supplied with the parameters; QSS3BucketName or QSS3KeyPrefix therefore it
+      # has ALWAYS been interpolated (regardless of product stack) to its default value which in this was:
+      # http://aws-quickstart.s3.amazonaws.com/quickstart-atlassian-services/submodules/quickstart-aws-vpc/templates/aws-vpc.template
+      #
+      # This is also the case for templates sourced from custom S3 buckets! 
+      # 
+      # If we change this template we need to make sure it will accommodate those product templates which are being called 
+      # from custom S3 buckets and do not pass the parameters; QSS3BucketName, QSS3KeyPrefix. 
+      #             
+      # If we dont do this stack deployments will fail! 
+      #             
+      # As a result we support both scenarios, the first condition, if met, will construct a non-region specific URL for those templates 
+      # that do not provide the mentioned parameters. The second condition will construct a region specific URL for those templates that
+      # do.
+      #
```

Comment:
In code comment will be removed once we have agreement on what is being done here.